h5p-bildetema-words-topic-image/language/.en.json

```
@@ -102,6 +102,15 @@
           "default": "Pause audio"
         }
       ]
+    },
+    {
+      "label": "Show written words label",
+      "default": "Show written words"
```

Should also use the boolean value
```suggestion
      "default": true
```